generators/new-wp-plugin/index.js

```diff
@@ -92,11 +91,33 @@ module.exports = generators.Base.extend({
       regex: 'lean-p',
       replacement: this.name
     }, settings) );
+    console.log( chalk.green('Completed!\n') );
   },
 
   install: function() {
     process.chdir( this.PLUGIN_FOLDER );
 
-    this.spawnCommandSync('composer', ['update']);
+    if ( fs.existsSync('./vendor') ) {
+      console.log( chalk.red('Composer has alredy been installed\n') );
+    } else {
+      this.spawnCommandSync('composer', ['install']);
```

Comment:
I don't see composer mentioned in #21 or #30.

As I understand they both do the same thing for a project with no composer.lock. And if we do have a composer.lock then we should run `composer update` only in this case (to update our compose.lock to the latest versions).

This article explains it, the flowchart part way down the page illustrates it nicely. https://adamcod.es/2013/03/07/composer-install-vs-composer-update.html